Cargo.lock

```diff
@@ -14,22 +14,19 @@ version = "0.1.0"
 dependencies = [
  "arch 0.1.0",
  "devices 0.1.0",
+ "epoll 4.0.1 (registry+https://github.com/rust-lang/crates.io-index)",
  "fc_util 0.1.0",
- "futures 0.1.18 (registry+https://github.com/rust-lang/crates.io-index)",
- "hyper 0.11.16 (registry+https://github.com/rust-lang/crates.io-index)",
  "kernel 0.1.0",
  "logger 0.1.0",
  "memory_model 0.1.0",
+ "micro_http 0.1.0",
  "mmds 0.1.0",
  "net_util 0.1.0",
  "rate_limiter 0.1.0",
- "serde 1.0.101 (registry+https://github.com/rust-lang/crates.io-index)",
- "serde_derive 1.0.101 (registry+https://github.com/rust-lang/crates.io-index)",
- "serde_json 1.0.41 (registry+https://github.com/rust-lang/crates.io-index)",
+ "serde 1.0.99 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
During rebase you reverted the `cargo update` from the v0.19.0 release PR.
You need to run a `cargo update` here.